nix/nim-status/build.nix

```diff
@@ -0,0 +1,122 @@
+{ pkgs, stdenv, lib, fetchFromGitHub
+# Dependencies
+, xcodeWrapper
+, androidPkgs
+, git 
+, platform ? "android"
+, arch ? "386"
+, api ? "23" }:
+
+let
+  osId = builtins.elemAt (builtins.split "\-" stdenv.hostPlatform.system) 2;
+  osArch = builtins.elemAt (builtins.split "\-" stdenv.hostPlatform.system) 0;
+
+  ANDROID_NDK_HOME = "${androidPkgs}/ndk-bundle";
```

Comment:
I'd rename that to `ndkBundle` since it isn't used as an env variable and the way it looks might confuse someone.